app/policies/app_group_policy.rb

```diff
@@ -35,6 +35,10 @@ def update_labels?
     barito_superadmin? || user.can_access_app_group?(record, roles: %i(admin owner))
   end
 
+  def update_redact_labels?
+    barito_superadmin? || user.can_access_app_group?(record, roles: %i(admin owner))
```

Comment:
let's keep for admin and owner first. please add this to notes and we might wanna get confirmation from prod engineer / service owner/

Reply:
Updated